modules/mig/main.tf

```diff
@@ -117,7 +117,7 @@ resource "google_compute_region_instance_group_manager" "mig" {
 }
 
 resource "google_compute_region_autoscaler" "autoscaler" {
-  provider = google
+  provider = google-beta
```

Comment:
Is this going to be a breaking change for existing clusters?  Please also verify if any updated is needed to the provider minimum version: https://github.com/terraform-google-modules/terraform-google-vm/blob/master/modules/mig/versions.tf#L26